testfiles/5ZoneAirCooled_ZoneAirMassFlowBalance.idf

```diff
@@ -120,7 +120,7 @@
   Timestep,4;
 
   ZoneAirMassFlowConservation,
-    Yes,                     !- Adjust Zone Mixing For Zone Air Mass Flow Balance
+    AdjustMixingOnly,        !- Adjust Zone Mixing and Return For Air Mass Flow Balance
```

Comment:
Manually transition example files.  There are two example files that require transition in the regression test suite.